livre/eq/Client.cpp

```diff
@@ -195,9 +250,20 @@ void Client::clientLoop()
 {
     do
     {
-         eq::Client::clientLoop();
-         LBINFO << "Configuration run successfully executed" << std::endl;
+        LBINFO << "Entered client loop" << std::endl;
+        const uint32_t timeout = 100; // Run idle function
+                                      // every 100ms
+        while( isRunning( ))
+        {
+            if( _impl->_idleFunc )
+                _impl->_idleFunc(); // order is important for latency
+
+            processCommand( timeout );
+        }
+
     }
-    while( _applicationParameters.isResident ); // execute at least one config run
+    while( _impl->_applicationParameters.isResident ); // execute at least one config run
```

Comment:
It is coming from veeeeeeery old Livre. While transferring code I haven't touched it. Maybe UZH guys are still using ?